portfolio/src/main/webapp/script.js

```diff
@@ -38,3 +38,27 @@ function addRandomStock() {
   const stockContainer = document.getElementById('stock-container');
   stockContainer.innerText = stock;
 }
+
+function getGreetingAndJsonUsingArrowFunctions() {
+  fetch('/data').then(response => response.text()).then((greeting) => {
+    document.getElementById('greeting-container').innerText = greeting;
+  });
+}
+
+//Fetches comments from the server and adds them to the DOM.
+function LoadComments(){
+    fetch('/data').then(response => response.json()).then((comments) => {
+        const commentListElement=document.getElementById('comment-list');
+        commentListElement.innerHTML=' ';
+        comments.forEach((comment)=>{
+            commentListElement.appendChild(createListElement(comment));
+        });
+    });
```

Comment:
nit: Let's un-indent this whole block by 2 spaces.

Reply:
Addressed